sql/moz-fx-data-shared-prod/firefox_ios_derived/attributable_clients_v1/query.sql

```diff
@@ -3,23 +3,52 @@ RETURNS INT64 AS (
   (SELECT SUM(value) FROM UNNEST(map))
 );
 
-WITH client_search_activity AS (
+WITH client_days AS (
   SELECT
     DATE(submission_timestamp) AS submission_date,
     sample_id,
     client_info.client_id,
+    LOGICAL_OR(mozfun.norm.extract_version(app_version, 'major') >= 108) AS has_search_data,
```

Comment:
How does extracting the app_version and checking it guarantees there is search data?

Reply:
Past version `108` on release, we were reporting search data on the baseline ping. Prior to 108, we were reporting it just on the metrics ping. This kind of check also handles clients who haven't upgraded yet: they will report e.g. `107` for the search values, so we will only get the search data from the metrics ping.